srcpkgs/git-interactive-rebase-tool/template

```diff
@@ -0,0 +1,12 @@
+# Template file for 'git-interactive-rebase-tool'
+pkgname=git-interactive-rebase-tool
+version=2.1.0
+revision=1
+archs="i686 x86_64"
```

Comment:
Why was this line added again?

Reply:
hmmm, I can remove it

Reply:
I think @ericonr had requested it for building on `i686`

Reply:
> There shouldn't be a reason to do this, but if there is, it should be clearly documented.

if it's not specified, the package will be built for all architectures. This way you restricted it to 2 architectures and excluded their musl variants.

Reply:
Oh @ericonr had requested to remove it also :)

Reply:
> > There shouldn't be a reason to do this, but if there is, it should be clearly documented.
> 
> if it's not specified, the package will be built for all architectures. This way you restricted it to 2 architectures and excluded their musl variants.

Is there a way I can test building for all architectures on my local machine with xbps-src?

Reply:
```
$ ./xbps-src -m masterdir.musl binary-bootstrap x86_64-musl
$ ./xbps-src -m masterdir.musl build git-interactive-rebase-tool
$ ./xbps-src -m masterdir.musl check git-interactive-rebase-tool
```